rest/utilities_testing.go

```diff
@@ -148,6 +148,20 @@ func (rt *RestTester) Bucket() base.Bucket {
 
 	rt.RestTesterServerContext = NewServerContext(&sc, false)
 
+	// Copy this startup config at this point into initial startup config
+	var initialStartupConfig StartupConfig
+	scBytes, err := base.JSONMarshal(sc)
+	if err != nil {
+		rt.tb.Fatalf("Unable to marshal initial startup config: %v", err)
+	}
+
+	err = json.Unmarshal(scBytes, &initialStartupConfig)
+	if err != nil {
+		rt.tb.Fatalf("Unable to unmarshal initial startup config: %v", err)
+	}
+
+	rt.RestTesterServerContext.initialStartupConfig = &initialStartupConfig
```

Comment:
Can we use `DeepCopyInefficient`? More clear about what's going on.

```suggestion
	// Copy this startup config at this point into initial startup config
	err := DeepCopyInefficient(rt.RestTesterServerContext.initialStartupConfig, sc)
	if err != nil {
		rt.tb.Fatalf("Unable to copy initial startup config: %v", err)
	}
```